src/atoms/dialog.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { Button, Dialog as MuiDialog, DialogActions, DialogContent, DialogContentText, DialogProps as MuiDialogProps, DialogTitle } from '@mui/material';
+
+interface DialogProps extends Omit<MuiDialogProps, 'content' | 'title'> {
+  actions?: React.ReactNode;
+  buttonType?: 'button' | 'submit'; 
+  content: React.ReactNode | string | undefined;
```

Comment:
`React.ReactNode` includes` string`. 
Could be only` React.ReactNode` instead of `React.ReactNode | string | undefined `